ekumamait/BookStore
# #167064107 setup app

**What does this PR do?**
- Add basic app setup
- Add express server
- Add SQLite database 

**Description of Task to be completed**
Create an express server that connects to an SQLite database

**How should this be manually tested?**
* Switch to the branch `ch-setup-app`
* Start the server using `npm start`
* See terminal logging `listening port` and `connection to database`

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
#167064107

**Screenshots (if appropriate)**
<img width="543" alt="Screen Shot 2019-10-10 at 11 20 03" src="https://user-images.githubusercontent.com/9946845/66556152-02e53e80-eb50-11e9-9ede-919776875d7c.png">
